Jellyfin/dist/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+  "author": {
+    "name": "alexbcberio",
+    "id": "202915432175239169"
+  },
+  "url": "jellyfin.org",
+  "description": {
+    "en": "The Free Software Media System"
+  },
+  "service": "Jellyfin",
+  "version": "1.0",
+  "logo": "https://raw.githubusercontent.com/jellyfin/jellyfin/master/Emby.Dlna/Images/logo240.png",
+  "thumbnail": "https://jellyfin.org/favicon.ico",
```

Comment:
Don't use github's cdn or their own, please upload the images to a site like imgur.

Reply:
the thumbnail is also a favicon xd, it should be an actual wide +500px image

Reply:
Updated in commit [5291b2347b892984d9236c93f73282aca2f9c483](https://github.com/alexbcberio/Presences/commit/5291b2347b892984d9236c93f73282aca2f9c483)